pkg/machines/components/networks/networkOverviewTab.jsx

```diff
@@ -0,0 +1,171 @@
+                <div id={`${idPrefix}-persistent`}> {network.persistent ? _("yes") : _("no")} </div>
+
+                <label className='control-label' htmlFor={`${idPrefix}-autostart`}> {_("Autostart")} </label>
+                <div id={`${idPrefix}-autostart`}> {network.autostart ? _("yes") : _("no")} </div>
+
+                { network.mtu && <React.Fragment>
+                    <label className='control-label' htmlFor={`${idPrefix}-mtu`}> {_("Maximum Transmission Unit:")} </label>
+                    <div id={`${idPrefix}-mtu`}> {network.mtu} </div>
+                </React.Fragment> }
+
+                { (qosInbound || qosOutbound) && <React.Fragment>
+                    <hr />
+                    <label className='control-label label-title'> { _("QoS") } </label>
+                    <span />
+
+                    { qosInbound && <React.Fragment>
+                        <label className='control-label' htmlFor={`${idPrefix}-bandwidth-inbound`}> {_("Inbound:")} </label>
+                        <div id={`${idPrefix}-bandwidth-inbound`}> {qosInbound} </div>
+                    </React.Fragment> }
+
+                    { qosOutbound && <React.Fragment>
+                        <label className='control-label' htmlFor={`${idPrefix}-bandwidth-outbound`}> {_("Outbound:")} </label>
+                        <div id={`${idPrefix}-bandwidth-outbound`}> {qosOutbound} </div>
+                    </React.Fragment> }
+                </React.Fragment> }
+            </div>
+
+            <div className="ct-form-layout">
+                { ip[0] && <React.Fragment>
+                    <hr />
+                    <label className='control-label label-title'> {_("IPv4 Address")} </label>
+                    <span />
+
+                    { ip[0].address && <React.Fragment>
+                        <label className='control-label' htmlFor={`${idPrefix}-ipv4-address`}> {_("Address:")} </label>
+                        <div id={`${idPrefix}-ipv4-address`}> {ip[0].address} </div>
+                    </React.Fragment> }
+
+                    { ip[0].netmask && <React.Fragment>
+                        <label className='control-label' htmlFor={`${idPrefix}-ipv4-netmask`}> {_("Netmask:")} </label>
+                        <div id={`${idPrefix}-ipv4-netmask`}> {ip[0].netmask} </div>
+                    </React.Fragment> }
+
+                    { ip[0].dhcp.range.start && <React.Fragment>
+                        <label className='control-label' htmlFor={`${idPrefix}-ipv4-dhcp-range`}> {_("DHCP range:")} </label>
+                        <div id={`${idPrefix}-ipv4-dhcp-range`}> {ip[0].dhcp.range.start + " - " + ip[0].dhcp.range.end} </div>
+                    </React.Fragment> }
+
+                    { ip[0].dhcp.hosts.map((host, index) => DHCPHost(host, index, ip[0].family, idPrefix))}
+                </React.Fragment> }
+
+                { ip[1] && <React.Fragment>
```

Comment:
Not ip[0]?

Reply:
No, from this line on, details about IPv6 (which is always on ip[1]) are shown.